android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -394,6 +402,46 @@ private void initializePlayer() {
         new Handler().postDelayed(new Runnable() {
             @Override
             public void run() {
+                YouboraLog.setDebugLevel(YouboraLog.Level.VERBOSE);
+
+                Options youboraOptions = new Options();
+                youboraOptions.setAccountCode(analyticsMeta.getString("accountCode"));
```

Comment:
Unfortunately calling set methods explicitly seems to be the only way for youbora to start recognising that the exoplayer is hooked up to it. These set methods have been used as per the confluence page

https://skynz.atlassian.net/wiki/spaces/SO/pages/5571129/Youbora+Implementation+-+Set+Youbora+Options?focusedCommentId=1565818922